tunnel.go

```diff
@@ -320,31 +320,20 @@ func (api *API) UpdateTunnelConfiguration(ctx context.Context, rc *ResourceConta
 	}
 
 	uri := fmt.Sprintf("/accounts/%s/cfd_tunnel/%s/configurations", rc.Identifier, params.TunnelID)
-	res, err := api.makeRequestContext(ctx, http.MethodPut, uri, params.Config)
```

Comment:
This is a slightly different bug but also has to do with the marshalling/unmarshalling of the `config` param.